src/driver/amdxdna/aie2_debugfs.c

```diff
@@ -453,6 +454,7 @@ static int aie2_telemetry(struct seq_file *m, u32 type)
 	if (!buff)
 		return -ENOMEM;
 
+	drm_clflush_virt_range(buff, size); /* device can access */
```

Comment:
Any reason why this is not required for other buffers alloc'ed through dma_alloc_noncoherent?

Reply:
This is required. See aie2_error.c. I just forgot to do clflush here. This is a bug fix.

Reply:
> This is required. See aie2_error.c. I just forgot to do clflush here. This is a bug fix.

OK.